foreman/data_refinery_foreman/surveyor/transcriptome_index.py

```diff
@@ -313,7 +313,7 @@ def _generate_files(self, species: Dict) -> None:
         fasta_download_url = url_builder.build_transcriptome_url()
         gtf_download_url = url_builder.build_gtf_url()
 
-        platform_accession_code = species.pop("division")
+        species.pop("division")
```

Comment:
It seems like we would have wanted this? Maybe bug?

Reply:
hmm not sure why we do this, looks like `species` is no longer used in this method after this.

Reply:
Hmm yeah this is odd, not sure why we'd call this platform_accession_code. It's also not used after the method either. Probably safe to remove the line entirely.

Reply:
and also the call below to `self._clean_metadata(species)`? Since `species` is no longer used here.

Reply:
Yes. My guess is we used more things from the species dict for the original file metadata before but now that's handled by the URL builder classes.

Reply:
ok removed `self._clean_metadata`. Will let the tests pass before merging.